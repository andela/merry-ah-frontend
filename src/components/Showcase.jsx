import React, { Component } from 'react';

export default class Showcase extends Component {
    render() {
      return (
        <section id="showcase">
            <div id="showcase_text">
                <div>
                    <h1>Bring Unspoken <br></br> Words to Life</h1>
                    <h4>ArtCave will deliver the best art gallery stories</h4>
                </div>
                <div>
                    <a href="#"><button type="button" class="showcase_button" >Get Started</button></a>
                </div>
            </div>
		</section>
      )
    }
  }
