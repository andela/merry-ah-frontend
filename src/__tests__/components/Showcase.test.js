import React from 'react';
import { shallow } from 'enzyme';
import Showcase from '../../components/Showcase';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<Showcase />);
});
describe('Showcase Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
