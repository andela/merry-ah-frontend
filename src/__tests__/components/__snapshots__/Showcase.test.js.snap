// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Showcase Component should match snapshot 1`] = `
<section
  id="showcase"
>
  <div
    id="showcase_text"
  >
    <div>
      <h1>
        Bring Unspoken 
        <br />
         Words to Life
      </h1>
      <div>
        <a
          href="#"
        >
          <button
            className="showcase_button"
            type="button"
          >
            Get Started
          </button>
        </a>
      </div>
    </div>
  </div>
</section>
`;
